cloudshell/shell/core/driver_context.py

```diff
@@ -65,13 +68,15 @@ def __init__(
         admin_auth_token,
         cloudshell_version,
         cloudshell_api_scheme,
+        use_webapi_endpoint=None,
     ):
         self.server_address = server_address
         self.cloudshell_api_port = cloudshell_api_port
         self.quali_api_port = quali_api_port
         self.admin_auth_token = admin_auth_token
         self.cloudshell_version = cloudshell_version
         self.cloudshell_api_scheme = cloudshell_api_scheme
+        self.use_webapi_endpoint = use_web_api_endpoint
```

Comment:
in the params and here different variable names - use_web_api_endpoint and use_webapi_endpoint (underscore in web api)

Reply:
thank you so much!!!
will now build the package and test to be sure it works properly